libs/gltfio/src/FilamentAsset.cpp

```diff
@@ -84,7 +84,9 @@ FFilamentAsset::~FFilamentAsset() {
         mEngine->destroy(ib);
     }
     for (auto tx : mTextures) {
-        mEngine->destroy(tx.texture);
+        if (UTILS_LIKELY(tx.shared == false)) {
+            mEngine->destroy(tx.texture);
+        }
```

Comment:
how do we know that other ones don't have `shared` to `true`?

Reply:
Because we know a priori that it's set up that way.  I had considered calling this field `isOwner` maybe that's a better name?

Reply:
I changed this field to `isOwner`, I think it's more clear now.